plat/ti/k3low/common/drivers/pm/include/psc.h

```diff
@@ -0,0 +1,363 @@
+/*
+ * Copyright (C) 2025 Texas Instruments Incorporated - https://www.ti.com
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/*
+ * FIXME: Perhaps remove off state, on state bumps use count,
+ * this would maintain dependencies as off could not be forced
+ */
+
+#ifndef PSC_H
+#define PSC_H
+
+#include <device.h>
+#include <lib/container_of.h>
+#include <pm_types.h>
+#include <device_limits.h>
+
+#define PSC_DEV_NONE		7U
+#define PSC_DEV_MULTIPLE	6U
+#if defined(CONFIG_PSC_PD_MAX_COUNT_64)
```

Comment:
Do we enable this config anywhere? If not can we plz delete this section from here and elsewhere?

```
$ rg 'CONFIG_PSC_PD_MAX_COUNT_64'                                         [16:23:50]
plat/ti/k3/common/drivers/pm/include/psc_soc_device.h
18:#if defined(CONFIG_PSC_PD_MAX_COUNT_64)

plat/ti/k3/common/drivers/pm/include/psc.h
22:#if defined(CONFIG_PSC_PD_MAX_COUNT_64)
34:#if defined(CONFIG_PSC_PD_MAX_COUNT_64)
111:#if defined(CONFIG_PSC_PD_MAX_COUNT_64)
```